src/foam/nanos/auth/User.js

```diff
@@ -289,14 +319,14 @@ foam.CLASS({
       }
     },
     {
-      class: 'DateTime',
+      class: 'Date',
       name: 'created',
-      documentation: 'Creation date'
+      documentation: 'Creation date.'
     },
     {
-      class: 'DateTime',
+      class: 'Date',
```

Comment:
Why did you change from DateTime to Date?  These are more operational and I've been using DateTime for all CreatedAware, LastModifiedAware so when the properties are displayed we do see the time properties.

Reply:
Should be DateTime.